server/index.py

```diff
@@ -0,0 +1,22 @@
+"""Server."""
+from flask import Flask
+from mimic.text_generator import TextGenerator
+from mimic.text_generator_factory import TextGeneratorFactory
+from flask import request
+import json
+app = Flask(__name__)
+
+
+@app.route('/test-endpoint', methods=['GET'])
```

Comment:
I think this could be named `/status` or something similar - `/test-endpoint` is a bit of a mouthful.